wrangler-core/src/main/java/io/cdap/directives/row/RecordConditionFilter.java

```diff
@@ -119,4 +122,13 @@ public Mutation lineage() {
     el.variables().forEach(column -> builder.relation(column, column));
     return builder.build();
   }
+
+  @Override
+  public List<WranglerMetric> getMetrics() {
```

Comment:
Is the same implementation repeating in multiple places? Do you want to make this as default implementation?

Reply:
afaik I believe we can't make this default since the information required for building metrics comes from implementation of the Directive.

Reply:
Yes, the required JEXL script is in the directive implementation